app/javascript/packs/assessments.js

```diff
@@ -3,8 +3,15 @@ Array.prototype.slice.call(document.querySelectorAll('.need--status_complete')).
 let showCompletedNeeds = false;
 const toggleDisplay = (show) => show ? 'table-row' : 'none';
 
-document.getElementById('toggle-visibility-completed-assessments').addEventListener('click', (e) => {
+const toggleVisibilityCompletedAssessments = document.getElementById('toggle-visibility-completed-assessments')
+toggleVisibilityCompletedAssessments.addEventListener('click', (e) => {
     showCompletedNeeds = !showCompletedNeeds;
+    const table = toggleVisibilityCompletedAssessments.parentNode.parentNode
```

Comment:
I think it's better to explicitly add semicolons going forward

Reply:
We have a lot js without `;` also `let` and `const` for example. We expect browser to support that.

Reply:
no I know the browser supports it, it's more for other issues can come out of it sometimes. I read an article about it some time ago, basically if you let js put the semicolons for you sometimes they get put in the wrong place, so it's better to add them explicitly

Reply:
You are right. It is good to put `;` on places you mention. For example around `return`. For all other places it should be project based.